python/lsst/sims/photUtils/Sed.py

```diff
@@ -481,6 +481,58 @@ def setFlatSED(self, wavelen_min=None,
         self.name = name
         return
 
+    def read_close_SED(self, teff, feH, logg):
+        """
+        Check the cached Kuruz models and load the model closest to the input stellar parameters
+        """
+        global _global_lsst_sed_cache
+
+        # Load the cache if it hasn't been done
+        if _global_lsst_sed_cache is None:
+            cache_LSST_seds()
+        # Build an array with all the files in the cache
+        if not hasattr(self, 'param_combos'):
+            kurucz_files = [filename for filename
+                            in _global_lsst_sed_cache if ('kurucz' in filename) &
+                            ('_g' in os.path.basename(filename))]
+            self.param_combos = numpy.zeros(len(kurucz_files),
```

Comment:
Makes sense to spin it off as a function. I think it's still a good idea to leave it in the `Sed.py` file since it's using the global variables in there.